### DISCUSSION TITLE
Question abut Dump1090 feed

### DISCUSSION THREAD

**Discussion by stetho** [ketilmo/balena-ads-b#61] [Q&A] [answered]
Hi,

On my FR24 (only) setup I used to be able to go to

http://fr24.home.lan/dump1090/data/aircraft.json

And get a JSON file of all the currently detected planes on my feeder. With this Balena image I'm now feeding aircraft to load of different services and they're all working fine but this Dump1090 feed is no longer exposed. I've gone as far as running NMAP against all Pi to find all the open ports and then prodding each one to see what responds but without any luck.

Am I looking in the wrong place or is this feed blocked in this Balena image?

Thanks

Steve

**Comment by ketilmo** [comment, marked as answer]
Hi Steve,

Try the following URL: http://DEVICE_IP/skyaware/data/aircraft.json

Let me know if that works for you.

Kind regards,
Ketil

**Reply by stetho** [reply]
Thanks - that's what I'm looking for.

**Reply by ketilmo** [reply]
Great! Happy to help. Thanks for reporting back.
